source/EvoPlanet/EvoPlanet.Server/Program.cs

```diff
@@ -36,12 +36,13 @@ public static void Main(string[] args)
             app.UseStaticFiles();
 
             // Configure the HTTP request pipeline.
+            /*
```

Comment:
This comment is also unnecessary. If you want to remove the code, delete it.

Also, block comments (/**/) show up worse in source control, than line comments (I see only two changed lines, but you effectively commented out five lines of code). Avoid block comments in commits, and try to avoid comments in PRs (Pull Requests).